br/pkg/storage/s3.go

```diff
@@ -428,16 +430,16 @@ func NewS3Storage(ctx context.Context, backend *backuppb.S3, opts *ExternalStora
 }
 
 // checkBucket checks if a bucket exists.
-func checkS3Bucket(svc *s3.S3, qs *backuppb.S3) error {
+func checkS3BucketCheck(_ context.Context, svc s3iface.S3API, qs *backuppb.S3) error {
```

Comment:
Why we need context.contex as input parameter, seems we do not use it.

Reply:
new added check use it